test/python/test_swap_eth_to_btc.py

```diff
@@ -61,6 +62,9 @@ def test_swap_eth_to_btc_wrong_amount(backend, firmware, navigator, test_name):
 def test_swap_eth_to_btc_ok(backend, firmware, navigator, test_name):
     amount = '013fc3a717fb5000'
     prepare_exchange(backend, firmware, navigator, test_name, amount)
+    sleep(1)
     eth = EthereumClient(backend, derivation_path=bytes.fromhex("058000002c8000003c800000000000000000000000"))
+    sleep(1)
     eth.get_public_key()
+    sleep(1)
```

Comment:
With the new ragger APIs for screen waiting, do we still need those sleeps ?

Reply:
Yes because nothing happens on the screen, we wait that Speculos has simulated the OS starting the other app

Reply:
I'm not sure, but I've had too many inconsistencies with eth in swap context and this sleep removes it